timelock-impl/src/test/java/com/palantir/atlasdb/backup/AtlasBackupResourceTest.java

```diff
@@ -63,8 +67,24 @@
     private final AsyncTimelockService mockTimelock = mock(AsyncTimelockService.class);
     private final AsyncTimelockService otherTimelock = mock(AsyncTimelockService.class);
 
-    private final AtlasBackupResource atlasBackupService =
-            new AtlasBackupResource(TARGETER, str -> str.equals("test") ? mockTimelock : otherTimelock);
+    private final AtlasBackupResource atlasBackupService = new AtlasBackupResource(
+            () -> BEARER_TOKEN, TARGETER, str -> str.equals("test") ? mockTimelock : otherTimelock);
+
+    @Test
+    public void prepareBackupThrowsIfAuthHeaderIsWrong() {
+        AuthHeader wrongHeader = AuthHeader.of(BearerToken.valueOf("imposter"));
+        assertThatServiceExceptionThrownBy(() -> AtlasFutures.getUnchecked(
+                        atlasBackupService.prepareBackup(wrongHeader, PREPARE_BACKUP_REQUEST)))
+                .hasType(ErrorType.PERMISSION_DENIED);
+    }
+
+    @Test
+    public void completeBackupThrowsIfAuthHeaderIsWrong() {
+        AuthHeader wrongHeader = AuthHeader.of(BearerToken.valueOf("imposter"));
+        assertThatServiceExceptionThrownBy(() -> AtlasFutures.getUnchecked(
+                        atlasBackupService.completeBackup(wrongHeader, completeBackupRequest(validBackupToken()))))
+                .hasType(ErrorType.PERMISSION_DENIED);
+    }
```

Comment:
is there a test for happy case?

Reply:
The other tests implicitly test the happy case :)